doc/sphinx/parameters/Adiabatic_20conditions_20model.md

```diff
@@ -2,7 +2,7 @@
 # Adiabatic conditions model
 
 
-## **Parameters in section** Adiabatic conditions model
+## **Subsection:** Adiabatic conditions model
```

Comment:
How is this shown in the latex manual? I think a better way to mark this up may be 
```suggestion
## Subsection &ldquo;Adiabatic conditions model&rdquo;
```